lingetic-nextjs-frontend/app/languages/[language]/QuestionListGrid.tsx

```
@@ -0,0 +1,30 @@
+  language,
+}: QuestionListGridProps) {
+  const router = useRouter();
+
+  return (
+    <div className="grid gap-4">
+      {questionLists.map((list: QuestionList) => (
+        <button
+          key={list.id}
+          onClick={() => router.push(`/languages/${language}/${list.id}`)}
+          className="bg-white p-6 rounded-xl shadow-lg hover:shadow-xl transition-shadow text-left w-full"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)